util/gpool/spmc/option.go

```diff
@@ -103,8 +103,8 @@ func loadTaskOptions(options ...TaskOption) *TaskOptions {
 	if opts.ResultChanLen == 0 {
 		opts.ResultChanLen = uint64(opts.Concurrency)
 	}
-	if opts.ResultChanLen == 0 {
-		opts.ResultChanLen = uint64(opts.Concurrency)
+	if opts.TaskChanLen == 0 {
+		opts.TaskChanLen = 1
```

Comment:
Could we use meaningful const instead of 0 or 1 to make code more readable?

Reply:
Done, add ```defaultTaskChanLen`` const.